### PR TITLE
re #707 Remove Hawtio navigation references, add and manage own routing

### PR DESCRIPTION
**Pull request by kahboom**
Many thanks to @msavy for helping me clean up this commit mess.

Here's where I messed up: Created a feature branch, worked off of it, created a feature branch off of that feature branch (once we decided to only remove Hawtio nav stuff, yet keep the original feature branch in tact in case I go back to it), ran `git pull upstream master` once or twice to further complicate matters, and then, because I wasn't satisfied with how much I had complicated things, merged it onto master, pushed and created a PR.

Lessons learned: Rebase master, don't pull from upstream. Do it once, at the end once I'm ready to merge. Create PRs for feature branches, not master. Maybe go a little less crazy with commits?

Original PR: https://github.com/apiman/apiman/pull/232

cc @EricWittmann

### CODE REVIEW

File: manager/ui/hawtio/gulpfile.js [hunk 1]
```diff
@@ -121,4 +121,4 @@ gulp.task('build', ['bower', 'path-adjust', 'tsc', 'template', 'concat', 'clean'
 gulp.task('default', ['connect']);
 
 
-    
+
```
Comment (kahboom) [inline manager/ui/hawtio/gulpfile.js]: Ugh this drives me nuts

File: manager/ui/hawtio/plugins/api-manager/ts/apimanPlugin.ts [hunk 2]
```diff
@@ -230,7 +374,7 @@ module Apiman {
                 next();
             });
         });
-        
+
     }, true);
 
     hawtioPluginLoader.addModule(Apiman.pluginName);
```
Comment (msavy) [inline manager/ui/hawtio/plugins/api-manager/ts/apimanPlugin.ts]: Is this one of the bits that's just staying in temporarily? Or did I accidentally bring that back in?

Comment (kahboom) [inline manager/ui/hawtio/plugins/api-manager/ts/apimanPlugin.ts]: Nope you're fine, that's one of the bits that's just staying in temporarily.

Comment (msavy) [inline manager/ui/hawtio/plugins/api-manager/ts/apimanPlugin.ts]: Phew :+1:

**Pull request merged**